website/docs/tutorials/react-native.md

```diff
@@ -217,4 +229,6 @@ The important point here is that the sentence isn't broken into pieces but remai
 
 -   [`@lingui/react` reference documentation](/docs/ref/react.md)
 -   [`@lingui/cli` reference documentation](/docs/ref/cli.md)
--   [Pluralization Guide](/docs/guides/plurals.md)
+-   [Localizing React Native apps talk from React Native EU 2022](https://www.youtube.com/live/uLicTDG5hSs?feature=share&t=7512)
+
+This guide originally authored and contributed in full by [Vojtech Novak](https://twitter.com/vonovak).
```

Comment:
hope you don't mind this; I'm self employed and this increases my "luck surface area" for finding new opportunities 🙂

Reply:
@vonovak don't see any issues with this 🙂